OpenRA.Mods.Common/Traits/RejectsOrders.cs

```diff
@@ -17,14 +17,19 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Can be used to make a unit partly uncontrollable by the player.")]
 	public class RejectsOrdersInfo : ConditionalTraitInfo
 	{
-		[Desc("Possible values include Attack, AttackMove, Guard, Move.")]
+		[Desc("Explicit list of rejected orders. Leave empty to reject all minus those listed under Except. ",
+			"Possible values include Attack, AttackMove, Guard, Move, Stop.")]
```

Comment:
Might be worth adding a mechanism to include order names as part of the autogenerated trait docs.